test/coverage/while.js

```diff
@@ -10,7 +10,7 @@ const internals = {};
 
 exports.method = function (value) {
 
-    while (value) {
+    while ( value ) {
```

Comment:
To make a difference between generated and original file, especially for chunks handling.